### PR TITLE
watchrezero2.com and related domains

### LINKED ISSUE #77764: watchrezero2.com
### Issue URL (Annoyance)
 [http://watchrezero2.com/rezero-season-2-episode-24-subbed-part-2-episode-11/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwatchrezero2.com%2Frezero-season-2-episode-24-subbed-part-2-episode-11%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Safari v1.8.10
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion
Other extensions used: | N/A

### PR DESCRIPTION
**Pull request by wout-codes**
#77764 and related domains.

### CODE REVIEW

**Comment by wout-codes** [comment]
WIP

**Comment by wout-codes** [comment]
@zloyden Finished.

**Review by zloyden — CHANGES_REQUESTED** [review]
> Finished.

Are you sure about it? Some sites have leftovers, clickunders.
For instance, `getpronews.com` is a dead domain.
`haikyuu3.com` has a clickunder.
`readopm.com` their social link is blocked by your rule `tokyoghoulre.com,watchgoblinslayer.com,readopm.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,getpronews.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img`. There is also a leftover.

So, please check them all again and change you description. It should contain links to problematic sites, the reason for adding domains to the rules.

**Comment by wout-codes** [comment]
@zloyden Thanks!
Can you help me out a little? It would take another about an hour if I review all myself.

**Comment by zloyden** [comment]
Ofc, I'll review it a bit later. There are many changes.

**Comment by wout-codes** [comment]
@zloyden What leftover, I can't seem to find it.

**Comment by zloyden** [comment]
https://ww3.readopm.com/
<details><summary>Leftover:</summary>

image

</details><br/>

**Comment by wout-codes** [comment]
@zloyden Is it okay if I add CSS Ext. for this? `readopm.com#?#.card:contains(Ad)`

**Comment by zloyden** [comment]
> Is it okay if I add CSS Ext. for this? `readopm.com#?#.card:contains(Ad)`

According their code, you can add `readopm.com##.widget-area > div.card:first-child`.

**Comment by wout-codes** [comment]
> > Is it okay if I add CSS Ext. for this? `readopm.com#?#.card:contains(Ad)`
> 
> According their code, you can add `readopm.com##.widget-area > div.card:first-child`.

Seems like I do better in Ext.CSS

**Review by zloyden — CHANGES_REQUESTED** [review]
```
watchsao.tv, watchgoblinslayer.com
```
Clickunder.
`#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")` it's ok for them.

**Comment by wout-codes** [comment]
> ```
> watchsao.tv, watchgoblinslayer.com
> ```
> 
> Clickunder.
> `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")` it's ok for them.

I couldn't reproduce it at the very first try, when I was collecting and adding the domains. Maybe due to cookies stored.

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -4110,7 +4109,7 @@ cloudstorageoptions.com##.adslot_left
 ||adee.hit.gemius.pl/gdejs/xgde.js$domain=sportacentrs.com
 ||webgate.tn/getlink/*.js
 ||webgate.tn/promo/*.gif
-techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img
+tokyoghoulre.com,watchgoblinslayer.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,getpronews.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:4112]: Remove `getpronews.com` from the rule.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
Now it looks ok on my end.

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt]: @zloyden Actually `abort-current-inline-script` breaks the whole `String.fromCharCode` function, as seen in the Console Log. It doesn't happen when I use uBO's `acis` in uBO.

**Pull request merged**